themes/stanford_basic/templates/field/field--su-policy-related.html.twig

```diff
@@ -62,7 +62,8 @@
     {% endif %}
     {% for item in items %}
       <article class="su-card" {{ item.attributes }}>
-      <h2 class='su-link su-card__link'>{{ item.content }}</h2>
+      <h2 class='su-link su-link--action su-card__link'>{{ item.content }}</h2>
+      <p>{{ items[loop.index-1].content['#options'].entity.body.summary }}</p>
```

Comment:
This should be done via the view settings or the display mode settings. I can't remember which

Reply:
@pookmish I'm going to need some guidance. I looked here `/admin/structure/display-modes/view` and there isn't a view for the related policies; it is block in the layout builder.